### ISSUE TITLE
Is ipyw.exe broken in 2.7.6.3 ?

### ISSUE THREAD

**Issue by andygraham** [IronLanguages/main#1545]
This may be me or it may be a problem with ipyw.exe - I don't know hence this post. I'm reaching the end of my project to build an IronPython GUI REPL with AutoComplete Intellisense  and other improvements to replace ipy.exe together with a lightweight IDE that can nevertheless edit and run programs also with AutoComplete Intellisense and other frilly bits I am used to in Visual Studio and finally a GUI based debugger that can be used with either, or with ipy.exe. The last step is to deciding and implementing how to get the IDE to produce programs to run standalone away from both REPL and IDE.

Taking this simple WinForms program called guitest.py and playing with ways to invoke it 

```import clr
clr.AddReference("System.Windows.Forms")
from System.Windows.Forms import *

class SimpleForm(Form):
    def __init__(self):
        self.Text = 'Simple Winform'

Application.Run(SimpleForm())
```
Running in ipy.exe works fine
\>\>\> execfile("C:\Users\Andy\Documents\IronPython Projects\Demos\guitest.py")

Compiling from the command line with ipyc.exe and running it works fine
C:\IronPython 2.7>ipyc /target:winexe "C:\Users\Andy\Documents\IronPython Projec ts\Demos\guitest.py"
C:\IronPython 2.7>guitest

Trying to run it from the same command line with ipyw.exe 
C:\IronPython 2.7>ipyw "C:\Users\Andy\Documents\IronPython Projects\Demos\guitest.py"
Hangs with an "IronPython Windows Console has stopped working" dialog with no meaningful problem details.

Any ideas anyone?

PS: Why am I bothering to do this? I tried PyDev but I don't like Eclipse and its workspaces and last I tried 2.7.6 seemed to have broken it. PTVS is fine but a bit heavy but too tied to the VS project structure for whipping up a quick bit of code and trying it out, and VS2013 is no longer supported. Notepad++ with syntax highlighting is fine but I really need Intellisense and wanted a more seamless experience so I decided to try to do for myself.

**Comment by simplicbe** [comment]
Have you tried `Visual Studio Code`? It also enables debugging Python Code and is very lightweight. I've also got IronPython Code debugged. But we'll take a look at `ipyw`. Thanks!

**Comment by andygraham** [comment]
Yes, I tried VS code but couldn’t get Intellisense to work in it. I commented on this in the old Gitter-Chat room and asked if anyone had succeeded but didn’t get any reply.

**Comment by slide** [comment]
I just tried this with the current 2.7.7 RC2 and it worked for me.  Can you check in the event viewer and see if there is anything about the crash?

**Comment by andygraham** [comment]
@slide 
Good call on the event viewer - I  found 

Application: ipyw.exe
Framework Version: v4.0.30319
Description: The process was terminated due to an unhandled exception.
Exception Info: System.IO.FileLoadException

I tried on a fresh IronPython 2.7.6.3 installation in a VM and it worked so I took a deep breath and uninstalled my main machine IroPython installation and reinstalled it and it now works. Very strange that ipy worked and ipyw didn't. I would have thought them so similar that both would break if something was amiss. Sorry for the false alarm :(

@simplicbe 
I tried VS code again and got the same problem as before.

With this in User Settings
 "python.pythonPath": "c:/IronPython27/ipy.exe"
I get a very long "loading" message when I try Intellisense but nothing else

However if I switch to CPython
  "python.pythonPath": "c:/Python27/python.exe"
I get full Intellisense

I guess that I am missing something if it works for you.
